pages/services/beta-dcos-monitoring/0.4.3-beta/operations/alertmanager/storage/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: layout.pug
+navigationTitle: Alert Manager Storage
+title: Alert Manager Storage
+menuWeight: 20
+excerpt: Alert Manager Storage
+render: mustache
+model: ../../../data.yml
+---
+
+#include /services/include/beta-software-warning.tmpl
+
+# Configuring Alert Manager Storage
+
+## DSS Volume
+
+You can configure Alert Manager to use a dedicated volume provided by the [DC/OS Storage Service](https://docs.mesosphere.com/services/beta-storage/0.5.3-beta/) (DSS).
+
+At least one volume must be available for Alert Manager to use. For more on creating volumes, see the [DSS documentation](https://docs.mesosphere.com/services/beta-storage/0.5.3-beta/cli-references/dcos-storage-volume/dcos-storage-volume-create).
+
+When configuring Alert Manager, select `MOUNT` for its volume `type`, choose the minimum `size` for the volume in MB, and enter the `profile` for the volume. For more on volume profiles, see the [DSS documentation](https://docs.mesosphere.com/services/beta-storage/0.5.3-beta/cli-references/dcos-storage-profile/).
+
+The following configuration will deploy Alert Manager with a `fast` DSS volume that is 512 MB or larger. If there is no such volume available, Alert Manager will fail to deploy.
```

Comment:
"The following configuration will deploy Alert Manager with a `fast` DSS volume that is 512 MB or larger."  change to: "The following configuration will deploy Alert Manager to a volume that has a volume profile of `fast` which is 512MB or larger.